src/MeetingsSDKAdapter.js

```diff
@@ -238,15 +245,22 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
    * @returns {Observable.<MeetingControlDisplay>}
    * @memberof MeetingJSONAdapter
    */
-  muteAudioControl(ID) {
+  audioControl(ID) {
     const sdkMeeting = this.fetchMeeting(ID);
     const muted = {
```

Comment:
The states seemed to be flipped. See [JSON implementation](https://github.com/webex/components/blob/master/src/adapters/MeetingsJSONAdapter.js) for reference